modules/openapi-generator/src/main/resources/kotlin-multiplatform-client/data_class.mustache

```diff
@@ -0,0 +1,50 @@
+import kotlinx.serialization.Required
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * {{{description}}}
+ *
+{{#allVars}}
+ * @param {{{name}}} {{{description}}}
+{{/allVars}}
+ */
+{{!TODO: Improve this?}}
```

Comment:
I have switch the array to list by default in KotlinClientCodegen.collectionType

Reply:
Yeah this also uses list now